ref/general/authorization.adoc

```diff
@@ -15,17 +15,18 @@
+In an application where only certain users or groups are entitled to access certain resources, users and groups are assigned roles that represent which resources they can access. For example, in a payroll application, users in the `Employee` role can view a page that lists their hourly wages and hours worked. However, only users in the `Manager` role can make changes to employee wages. During link:/docs/ref/general/#authentication.html[authentication], Open Liberty determines who a user is and what roles they are in. During authorization, Open Liberty determines how these roles correspond to different protected resources on the server.
+
+The roles that users and groups are checked against can either be defined directly in the application or applied to the application by way of the server configuration. It is also possible to avoid defining roles altogether if you specify role names that match the group names in the user registry exactly. In this case, the server  maps the role to the group with a matching name by default. In this configuration, you do not specify an `<application-bnd>` element in the `server.xml` file.
 
-The roles that users and groups are checked against can either be defined directly in the application or applied to the application by way of the server configuration.
 Roles are defined directly in the application by setting the `@DeclareRoles` annotation or by using `<security-role>` elements in the `web.xml` deployment descriptor file.
 To apply roles to an application from the server configuration, specify the `<security-role>` element within the `<application-bnd>` element in the `server.xml` file.
 
-It is also possible to avoid defining roles altogether if you specify role names that match the group names in the user registry exactly. In this case, the server automatically maps the role to the group with a matching name. In this configuration, you do not specify an `<application-bnd>` element in the `server.xml` file. 
+In some use cases, an application needs to know only whether a user is authenticated or not. In such cases, configuration can be simplified by specifying `"**"` as the role name in the `@RolesAllowed` annotation or XML configuration. This value denotes that any authenticated user can access a resource. It is also possible to effectively bypass authorization altogether by using the `@PermitAll` annotation, which specifies that a resource is accessible to everyone, even users who are not authenticated.
```

Comment:
PermitAll doesn't really bypass authorization, it is a kind of authorization that allows anyone. I know the distinction might seem nit picky, but I think it is better to think of PermitAll as an authorization rule that is very permissive rather than not having any.